guides/how_to_use_2D_velocity_estimate.ipynb

```diff
@@ -19,9 +19,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
We could consider configuring the notebook with nbdime: https://nbdime.readthedocs.io/en/latest/
This would make git diff results readable. I guess it depends on whether we will include a lot of jupyter notebooks in this repo.